crates/db_views/src/post_view.rs

```diff
@@ -365,17 +365,17 @@ impl<'a> PostQuery<'a> {
     };
 
     if self.saved_only.unwrap_or(false) {
-      query = query.filter(post_saved::id.is_not_null());
```

Comment:
I hate SQL sometimes. Even though the the join comes back null, and shouldn't even really have to check any column, it did anyway. Changing this to one of the joinable columns made it run instantly.